remind_class.py

```diff
@@ -0,0 +1,77 @@
+import email, smtplib, ssl
+from email.message import EmailMessage
+import time 
+from datetime import datetime
+
+################################################################################
+def look_up_gateway():
+    gate_way = {'AirFire Mobile': "number@sms.airfiremobile.com", 'Aio Wireless': "number@mms.aiowireless.net", 
+    'Alaska Communications': "number@msg.acsalaska.com", 'Alltel (Allied Wireless)' : "number@mms.alltelwireless.com", 'Verizon': "number@vzwpix.com.",
+    "Assurance Wireless": "number@vmobl.com", "AT&T": "number@mms.att.net", "BellSouth": "number@bellsouth.cl", "Bluegrass Cellular": "number@mms.myblueworks.com",
+    'Bluesky Communications': "number@psms.bluesky.as", 'Boost mobile': "number@myboostmobile.com", 'CellCom': "number@cellcom.quiktxt.com", 'Cellular South':"number@csouth1.com",
+    'Centennial Wireless': "number@cwemail.com", 'Chariton Valley Wireless': "number@sms.cvalley.net", 'Chat Mobility': "number@mail.msgsender.com",
+    'Cincinnati Bell': "number@mms.gocbw.com", 'Cleartalk': "number@sms.cleartalk.us", 'Cricket': "number@mms.cricketwireless.net", 'C Spire Wireless': "number@cspire1.com", 
+    'Edge Wireless': "number@sms.edgewireless.com", 'Element Mobile': "number@SMS.elementmobile.net", 'General Communications Inc.': "number@mobile.gci.net", 
+    'Golden State Cellular': "number@gscsms.com", 'GreatCall': "number@vtxt.com", 'MetroPCS': "number@mymetropcs.com", 'Pocket Wireless': "number@sms.pocket.com", 'Red Pocket Mobile':"number@vtext.com"}   
+    key = input('What carrier do you use: ')
+    if key in gate_way:
+      return gate_way[key]
+    else:
+      print("Sorry, carrier not found.")
+
+def email_alert(subject,body, to):
+    msg = EmailMessage()
+    msg.set_content(body)
+    msg['subject'] = subject
+    msg['to'] = to
+
+    user = "reminderbot975@gmail.com"
+    msg['from'] = user
+    password = "wnbpnfqsnhmgmahs"
+
+    server = smtplib.SMTP("smtp.gmail.com", 587)
+    server.starttls()
+    server.login(user, password)
+    server.send_message(msg)
+
+    server.quit()
+
+#used to ensure a working value is entered, when setting the time wanted for reminder
+def valid_time(reminder_time):
+    if len(reminder_time) != 11:
+        return "Invalid time! Please try again..."
+    else:
+        if int(reminder_time[0:2]) > 12:
+            return "Invalid HOUR! Please try again..."
+        elif int(reminder_time[3:5]) > 59:
+            return "Invalid MINUTE! Please try again..."
+        elif int(reminder_time[6:8]) > 59:
+            return "Invalid SECOND! Please try again..."
+        else:
+            return "valid"
```

Comment:
Functions often return `True` or `False`, which are boolean values. These are easier to interpret by other functions as opposed to strings like "valid". 

If you want to print warning message and stop the program, look into 
```
raise Exception('There is a problem....')
```

https://realpython.com/python-exceptions/#raising-an-exception